.travis.yml

```diff
@@ -1,4 +1,6 @@
+# use ubuntu 20.04 to get up to date python3 and node
+dist: focal
+
```

Comment:
Nit: I think current Travis images have Node v14, and we don't use Python in this package.

Reply:
(I meant that we don't have to specify the dist at all, but not a big deal)